README.md

```diff
@@ -13,10 +13,19 @@ The Gini Vision Library can be integrated in two ways, either by using the *Scre
 
 Customization of the Views is provided mostly via overriding of app resources: dimensions, strings, colors, texts, etc. Onboarding can also be customized to show your own pages, each consisting of an image and a short text.
 
-The Gini Vision Library has been designed for portrait orientation. We recommend limiting your concrete Activities which extend the Screen API's abstract Activities to portrait orientation. In case you use the Component API, you should limit the Activities hosting the Component API's Fragments to portrait orientation.
+The Gini Vision Library can be used on smartphones and tablets, too. On smartphones it has been designed for portrait orientation only and will always switch to portrait orientation in both Screen API and Component API usage. On tablets both portrait and landscape orientations are supported.
+
+It is not required to limit your Activities extending the Screen API's abstract Activities or your Activities hosting the Component API's Fragments to portrait orientation. The Gini Vision Library takes care of limiting to portrait on smartphones.
 
 Due to in-memory image handling applications using the Gini Vision Library must enable large heap.
 
+Tablet Support
+--------------
+
+We have adapted some UI elements to offer a better experience to tablet users and removed the camera flash requirement for tablets since flash is not present on all tablets. For more information please consult our guide [Supporting Tablets](http://developer.gini.net/gini-vision-lib-android/html/supporting-tablets.html).
+
+> **Note:** Due to the lower resolution cameras and lack of LED flash on some tablets the extraction quality might be lower compared to smartphones.
```

Comment:
I would change this note to perhaps the following: "Please see our minimum hardware recommendations for tablets below. We recommend implementing corresponding hardware checks for the GVL to deliver optimal results to users. Allowing tablets that do not meet the minimum hardware recommendations to use the GVL functionality could lead to lower extraction quality and longer processing times."

Reply:
The thing is, that even meeting our min tablet requirements does not guarantee same extraction quality as on phones. We don't require flash, which would make the extraction quality similar, but would lock out a lot of tablets that would meet all requirements, but flash. Flash is not standard for some reason on tablets.

I would use the following text: `Please see our minimum hardware recommendations for tablets below. We recommend implementing corresponding hardware checks for the GVL to deliver optimal results to users. Since many tablets with at least 8MP cameras don't have an LED flash (like the popular Samsung Galaxy Tab S2) we don't require flash for tablets. For this reason the extraction quality on those tablets might be lower compared to smartphones.

Reply:
👍